docs/source/cluster_management/clis.rst

```diff
@@ -48,6 +41,16 @@ timestamp
 
 Read or recalculate the immutable timestamp. Run ``./bin/atlasdb help timestamp`` for more information.
 
+kvs migration
+-------------
+This cli can help you migrate your AtlasDB client product from one KVS to another.  You will need to supply two different KVS configurations to the script.  In the general case you first call –setup, then –migrate, then –validate each time supplying the old and new configurations.  For more information run ``./bin/atlasdb help migrate`` for more information.
+ 
```

Comment:
Probably you want either --setup/--migrate/--validate or to remote the '-'s entirely.